flink-contrib/flink-storm-compatibility/flink-storm-compatibility-core/src/test/java/org/apache/flink/stormcompatibility/wrappers/StormBoltWrapperTest.java

```diff
@@ -222,6 +239,31 @@ public void testOpenSink() throws Exception {
 
 	@SuppressWarnings("unchecked")
 	@Test
+	public void testOpenWithStormConf() throws Exception {
+		final IRichBolt bolt = mock(IRichBolt.class);
+		final StormBoltWrapper<Object, Object> wrapper = new StormBoltWrapper<Object, Object>(bolt);
+
+		Configuration jobConfiguration = new Configuration();
+		jobConfiguration.setString(new String("path"), new String("/home/user/file.txt"));
+		jobConfiguration.setInteger(new String("delimitSize"), 1024);
+		Environment env = new RuntimeEnvironment(new JobID(), new JobVertexID(), new ExecutionAttemptID(),
+				new String(), new String(), 1, 2, jobConfiguration, mock(Configuration.class), mock(ClassLoader.class),
+				mock(MemoryManager.class), mock(IOManager.class), mock(BroadcastVariableManager.class),
+				mock(AccumulatorRegistry.class), mock(InputSplitProvider.class), mock(Map.class),
+				new ResultPartitionWriter[1], new InputGate[1], mock(ActorGateway.class),
+				mock(TaskManagerRuntimeInfo.class));
+		StreamingRuntimeContext ctx = new StreamingRuntimeContext(env, new ExecutionConfig(),
+				mock(KeySelector.class),
+				mock(StateHandleProvider.class), mock(Map.class));
+
+		wrapper.setup(mock(Output.class), ctx);
+		wrapper.open(mock(Configuration.class));
```

Comment:
The open() is usually called by openAllOperators(), and the Configuration config parameter is usually task Configuration, not job Configuration. So i mock it.